### PR TITLE
#1490: PM: add confirmation prompt to uninstall or upgrade package

### PR DESCRIPTION
**Pull request by MikhailArkhipov**

### CODE REVIEW

**Comment by MikhailArkhipov** [comment]
Also #1486

File: src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs [hunk 1]
```diff
@@ -184,6 +185,13 @@ internal class RPackageManagerViewModel : BindableBase, IRPackageManagerViewMode
 
         private async Task UninstallAsync(IRPackageViewModel package) {
             _coreShell.AssertIsOnMainThread();
+
+            if(MessageButtons.No == _coreShell.ShowMessage(string.Format(CultureInfo.InvariantCulture,
```
Comment (AlexanderSher) [inline src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs]: We need similar check for Update

Comment (MikhailArkhipov) [inline src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs]: Done

**Comment by AlexanderSher** [comment]
:+1:

**Pull request merged**